packages/nodes-base/credentials/TodoistApi.credentials.ts

```diff
@@ -16,4 +18,17 @@ export class TodoistApi implements ICredentialType {
 			default: '',
 		},
 	];
+	authenticate: IAuthenticateHeaderAuth = {
+		type: 'headerAuth',
```

Comment:
Should better use special built-in type "bearer"

Reply:
had to use function here as request fails if it contains empty body

Reply:
Not sure I understand what do you mean with "function"? The type "headerAuth" and "bearer" should theoretically work 100% identical and an empty body should work or fail for both.

Reply:
it did actually fails for both, I did replace auth with a function and removing empty body from request, otherwise test resolves to an error 
![image](https://user-images.githubusercontent.com/88898367/168820842-9d35887d-b29f-4c9a-ad4f-4a5d78db5433.png)